meetings/2022-09-08.md

```diff
@@ -44,51 +46,72 @@
-
-### Add Solid Notification JSON-LD context
-URL: https://github.com/solid/vocab/pull/62
-
->SC: We need to follow up on this after resolving some issues in solid/notifications (linked in the review). It should be published this alongside the next release of the Notifications Protocol.
-
-
-
-### Interchangeable use of notification channel type and subscription type
-URL: https://github.com/solid/notifications/issues/71
-
-
-### Define Notification Content
-URL: https://github.com/solid/notifications/pull/60
-
-* SC: Not critical for the next release but the underlying intent would be useful for the spec.
-* SC: At the very least, right now it highlights what may be overspecifying, constraining, exposing information.
-* SC: We should consider a simpler / minimal model for conformance.
-* SC: (I suggested a way that can beNotification Profiles
-
-### Notification Profiles
-URL: https://github.com/solid/notifications/issues/101
-
->Introduce the notion of Notification Profile as an extension of the core Notification Data Model.
->Specific Notification Profiles can then be independently published.
-
-
-### Solid Notifications Protocol: Review notes
-URL: https://github.com/solid/notifications/issues/76
-
->ACTION: AC to respond to design re performance of making the connection for live updates.
-
-* SC: I wanted to respond to that but didn't get around to it.
-
-
-### Cases where Client might want to update Subscriptions/Connections
-URL: https://github.com/solid/notifications/issues/103
-
-* SC: Action from https://github.com/solid/notifications-panel/blob/main/meetings/2022-08-18.md#an-alternative-websockets-design :
-
->ACTION: RG to create issue to note the cases in which client may want to update its subscriptions or connections.
-
-* SC: Before discussing that issue, does https://github.com/solid/notifications/issues/97 need a follow-up? Does 103 supersede 97? Close 97?
+* SC: Notification Channel Resource is the current term for the resource that includes some information about the notification channel/service and its capabilities.
+* SC: Some people find `http://www.w3.org/ns/solid#storageDescription` useful - at least on the Solid Protocol level. Current ED explains how to take advantage of it.. in addition to `describedby`. Thoughts?
+* RG: Is it two different mechanisms available all the time?
+* SC: Same same different. resource-specific or storage-wide cases.
+* RG: But in the end it links to two places to get that information.
+* AC: Asking for clarity: in the link header, is all of the information in the header available or ?
+* RG: No. Will the info be in describedby or the storageDescripiton. How would they know about the storage description when they access the first resource.
+* SC: examples:
+
+```http
+HEAD /foo
+
+Link: </notification-resource>; rel="describedby"
+Link: </notification-resource>; rel="http://www.w3.org/ns/solid#storageDescription"
+```
```

Comment:
Lines 69 thru 74 should be indented 2 space chars, such that they render as "inside" SC's comment (line 67).  GitHub won't let me suggest this change, because it doesn't handle embedded code fences properly.

Reply:
Resolved as per https://github.com/solid/notifications-panel/pull/70/commits/a40f782b82454a45238b5554adb674998bf5d371